py/vttest/environment.py

```diff
@@ -28,7 +28,14 @@
 
 def get_test_directory():
   """Returns the toplevel directory for the tests. Might create it."""
-  directory = tempfile.mkdtemp(prefix='vt')
+  dataroot = os.environ['VTDATAROOT']
+  if dataroot:
+    directory = os.path.join(dataroot, "test")
```

Comment:
That was my first thought. But then, I felt that 'test' was more explicit. Ideally, generating a unique name would be awesome, but I couldn't find a good utility for it.
Or, how about 'local'?